common/authen.js

```diff
@@ -123,18 +125,19 @@
                         keyboard: false
                     });
 
-                   
+
 
                     /* if browser is IE then add explicit handler to watch for changes in localstorage for a particular
                      * variable
                      */
                     if (UriUtils.isBrowserIE()) {
 
-                        
+
                         $cookies.put("chaise-" + referrerId, true, { path: "/" });
                         var intervalId;
                         var watchChangeInReferrerId = function () {
                             if (!closed && !$cookies.get("chaise-" + referrerId)) {
+                                // TODO: Attach .catch w/ catchAll
```

Comment:
This is unnecessary because the outer `.catch()` that should be added to [line 181](https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/authen.js#L182) will catch an error from here.